seeds/trials.js

```diff
@@ -479,6 +484,55 @@ exports.seed = (knex) => {
     },
   ];
 
+  const riskOfBiasCriterias = [
+    {
+      id: '756b06ca-8414-11e6-a6fa-e4b3181a2c8c',
+      name: '3.1',
+      created_at: new Date('2016-01-02'),
+      updated_at: new Date('2016-02-01'),
+    },
+    {
+      id: '7036b016-841c-11e6-a6fa-e4b3181a2c8c',
+      name: '3.2',
```

Comment:
I remember we agreed to keep this because the natural language formulation might change more often and we don't want to run migrations every time that happens. We can keep a mapping in the model, but I can also change it to natural language if necessary.